test/hotspot/jtreg/serviceability/sa/TestJhsdbJstackMixedWithXComp.java

```diff
@@ -32,14 +33,36 @@
 import jdk.test.lib.process.OutputAnalyzer;
 
 /**
- * @test
+ * @test id=xcomp
  * @bug 8370176
  * @requires vm.hasSA
  * @requires os.family == "linux"
  * @requires os.arch == "amd64"
```

Comment:
Can't we run the test on AArch64 now?

Reply:
I think this PR works on Linux AArch64, but I'm not sure because previous change (#27885) worked on some Linux AArch64, but there are some platforms whch did not work. Thus I think it is better to keep this test for Linux AMD64 only.

I couldn't reproduce the problem on Fedora Rawhide on Raspberry Pi 4, but there might be some issue because SA does not have DWARF parser for AArch 64 - it means the native we cannot find sender frame which depends on DWARF, does not use FP as frame pointer.